services/def.json

```diff
@@ -0,0 +1,32 @@
+{
+    "roles":
```

Comment:
Why "roles" instead of "classes" or "extra_classes"? I didnt see any special role_ prefix or meta tags attached to the classes defined from there. Maybe it would make more sense if the classes were prefixed with role_ and tagged?